include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -283,6 +283,15 @@ struct __parallel_merge_submitter_large<_IdType, _CustomName,
         });
     }
 
+    template <typename _Rng1, typename _Rng2, typename _Compare>
+    inline static _split_point_t<_IdType>
+    __find_start_point_w(const _Rng1& __rng1, const _Rng2& __rng2, const _split_point_t<_IdType>& __sp_left,
```

Comment:
What's a purpose of `__find_start_point_w`? It does nothing additional things, beside re-calling `__find_start_point`...